src/Symfony/Component/Messenger/Command/FailedMessagesRetryCommand.php

```diff
@@ -158,7 +160,9 @@ private function runInteractive(SymfonyStyle $io, bool $shouldForce)
 
     private function runWorker(ReceiverInterface $receiver, SymfonyStyle $io, bool $shouldForce): int
     {
-        $listener = function (WorkerMessageReceivedEvent $messageReceivedEvent) use ($io, $receiver, $shouldForce) {
+        $count = 0;
+        $listener = function (WorkerMessageReceivedEvent $messageReceivedEvent) use ($io, $receiver, $shouldForce, &$count) {
+            ++$count;
```

Comment:
Is `$count` needed at all anymore?

Reply:
yes to see if a message was processed as it was before